src/components/SingleItem.js

```diff
@@ -1,55 +1,126 @@
+import Link from 'next/link';
 import styled from 'styled-components';
 
 import useStore from './../hooks/useStore';
 
+const Wrapper = styled.section`
+	padding: 24px;
+`;
 const Picture = styled.img`
 	max-width: 185px;
 	border-radius: 16px 16px 16px 16px;
 	box-shadow: 1px 1px 10px 5px rgba(0, 0, 0, 0.2);
 `;
 
-const MovieHeadline = styled.p`
-	padding: 12px 8px 0 16px;
-	color: black;
+const Hl2 = styled.h2`
```

Comment:
I prefer more telling names for components, even if they're just a humble headline. A shorthand like `Hl2` could be hard to understand if seen out of context or in the future.

Reply:
I was going away from that, simply because the detail pages (movies and tv) has so many differences and small things that it made more sense for me to have the headlines numbered 2, 3 and 4. 😄

Reply:
But couldn't you have them numbered and give them ab proper name? Like `<Headline1>`, `<Headline2>`, …